hack/generator/pkg/astmodel/testdata/ConvertBetweenAliasAndAliasType.golden

```diff
@@ -20,7 +20,7 @@ func (person Person) ConvertFromVNext(source *vNext.Person) error {
 }
 
 // ConvertToVNext populates the provided destination Person from our Person
-func (person Person) ConvertToVNext(destination *vNext.Person) error {
+func (person *Person) ConvertToVNext(destination *vNext.Person) error {
```

Comment:
No sure about this one - is it just for consistency? I guess they might be quite big.